packages/hardhat-etherscan/src/index.ts

```diff
@@ -124,6 +131,10 @@ const verify: ActionType<VerificationArgs> = async (
     }
   );
 
+  const libraries: Libraries = await run(TASK_VERIFY_GET_LIBRARIES, {
```

Comment:
This isn't used anywhere. Plus, it has a `Libraries` type, but the task can return undefined.

Reply:
Maybe this is the reason it's not working.

Reply:
Indeed. I missed this completely. I was about to test it manually since it's a bit impractical to write a test just for this.